rir/src/common.h

```diff
@@ -4,8 +4,7 @@
 #include <cassert>
 #include <cstdint>
 
-// TODO force inlining for clang & gcc
-#define RIR_INLINE __attribute__((always_inline)) inline
+#define RIR_INLINE inline
```

Comment:
Yes. According to this: http://gcc.gnu.org/onlinedocs/gcc/Inline.html, we should also start moving macros to inline functions

Reply:
I fear we will actually regress a bit over this. but I was getting sick of figuring out where to put the RIR_INLINE's and in the end just making it worse. So lets try this for once and then see what happens.